docs/pages/setup/helm-reference/teleport-cluster.mdx

```diff
@@ -63,13 +63,26 @@ This reference details available values for the `teleport-cluster` chart.
 
 `kubeClusterName` sets the name used for the Kubernetes cluster. This name will be shown to Teleport users connecting to the cluster.
 
-## `authenticationType`
+## `authentication`
+
+### `authentication.type`
```

Comment:
I think all field-specific headings are H2s.
```suggestion
## `authentication.type`
```

Reply:
I checked and we use h3 in the reference for nested values, I'll correct all other titles